data_summaries/Deita 10K.json

```diff
@@ -0,0 +1,61 @@
+{
+    "deita-10k": {
+        "Unique Dataset Identifier": "deita-10k",
+        "Dataset Name": "deita-10k",
+        "Paper Title": "What Makes Good Data for Alignment? A Comprehensive Study of Automatic Data Selection in Instruction Tuning",
+        "Dataset URL": "https://huggingface.co/datasets/hkust-nlp/deita-10k-v0",
+        "GitHub URL": "https://github.com/hkust-nlp/deita",
+        "Hugging Face URL": "https://huggingface.co/datasets/hkust-nlp/deita-10k-v0",
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2312.15685",
+        "Semantic Scholar Corpus ID": 266551413,
+        "Collection": "Deita 10K",
+        "Collection URL": "https://huggingface.co/collections/hkust-nlp/deita-6569c198c174808d94cf5bd4",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Alignment",
+            "Language Modeling"
+        ],
+        "Text Sources": [],
+        "Format": [
+            "Zero-shot",
+            "Multi-turn Dialog"
+        ],
+        "Dataset Filter IDs": [
+            "ShareGPT",
+            "UltraChat",
+            "WizardLM"
```

Comment:
Each of these 3 data collections are separately in the Data Provenance Collection data summaries. Could you take a peak at them and add the Text Sources, Task Categories, Licenses, and Languages you see in them? Since they make up this dataset, all those things are represented in here unless Deita says they explicitly filter them out :)